Tools/cases_generator/analyzer.py

```diff
@@ -1,9 +1,10 @@
-from dataclasses import dataclass
-import lexer
-import parser
 import re
+from dataclasses import dataclass
 from typing import Optional
 
+import lexer as lx
```

Comment:
Is there any reason for a rename? Looks like it produces quite a lot of changes.

Reply:
1. To be consistent with the other files (other files import the lexer as lx).
2. To reduce the line widths.

Reply:
(mostly 1 if you want a real reason)

Reply:
I reverted this change (everytime a new commit is done in the file, I need to resolve conflicts, so let's keep it simple)